recipes-devtools/firmware-package-tool/files/iot2050-fwu-update-json.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+#
+# Copyright (c) Siemens AG, 2023
+#
+# Authors:
+#   Li Hua Qian <huaqian.li@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+update_json()
+{
+    sed -i '/"version": ".*"/s/"V.*"/"'$2'"/g' $1
+    sed -i '/"description": ".*"/s/V.*["$]/'$2\"'/g' $1
+}
+
+generate_fwu_tarball()
+{
+    echo "Generating the firmware tarball..."
+
+    if [ ! -e $2/iot2050-pg1-image-boot.bin ]; then
```

Comment:
Anyway we should build the pg1 and pg2 uboot previously. 
this is not a full automatic build flow.
Could it be done through our pipeline?

Reply:
Agreed, to build the pg1 and pg2 previously is better. 

Here is a workaround after internal discussion. Since @jan-kiszka  is also not agreed with this solution, I guess we need to discuss more.